keyboards/cipulot/ec_23u/keymaps/via/rules.mk

```diff
@@ -1,3 +1,3 @@
 VIA_ENABLE = yes
 
-SRC += via_apc.c
+SRC += keyboards/cipulot/common/via_ec.c
```

Comment:
This could probably be better handled with post_rules.mk at the kb level, and: 
```make
ifeq ($(strip $(VIA_ENABLE)), yes)
    SRC += keyboards/cipulot/common/via_ec.c
endif
```

Reply:
Committed with 12991cd and tested working.